### ISSUE TITLE
IMS data file creation

### ISSUE THREAD

**Issue by gilbertd26** [laiadc/PFM_Bearing_Fault_Detection#2]
how to create IMS data Test1.csv and Test2.csv

**Comment by lywsyx4399** [comment]
Is the app mentioned in the article public?

**Comment by laiadc** [comment]
No, sorry, the application is proprietary as it was developed as part of my master's thesis project in collaboration with a company.

**Comment by lywsyx4399** [comment]
Ok, thanks for your reply.
